tvx/dummy_testingt.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+var t dummyTestingT
```

Comment:
collapse back to the one in drivers?

Reply:
This tests do use `require` and `asserts` in the `main` package :/ If they are not too many, I will convert them.

Reply:
They are not too many, but probably best to do that in one go after we transition the tests... no point converting `assertEqual` in one file, but keeping it in 10 others.

Reply:
Falling back to drivers.T, although it is similarly ugly... :/